### PR TITLE
Fixing issue #11 for 1.x.x branch.

### PR DESCRIPTION
**Pull request by KatseNarniaan**
Basically the same changes as cjelger's commits for issue 11, but for the 1.x.x branch.

### CODE REVIEW

File: src/main/java/io/openapitools/swagger/OutputFormat.java [hunk 1]
```diff
@@ -40,9 +41,11 @@ public void write(Swagger swagger, File file) throws IOException {
     static class JSONWriter implements SwaggerWriter {
 
         @Override
-        public void write(Swagger swagger, File file) throws IOException {
-            ObjectMapper mapper = new ObjectMapper();
-            mapper.setSerializationInclusion(JsonInclude.Include.NON_EMPTY);
```
Comment (langecode) [inline src/main/java/io/openapitools/swagger/OutputFormat.java:45]: I think you change the semantics here by not keeping the `NON_EMPTY` option?

**Comment by langecode** [comment]
Thank you very much for your contribution. On small comment around the removal of `NON_EMPTY` option.

File: src/main/java/io/openapitools/swagger/OutputFormat.java (hunk 1 above)
Comment (KatseNarniaan) [reply src/main/java/io/openapitools/swagger/OutputFormat.java:45]: It is actually from here: https://github.com/openapi-tools/swagger-maven-plugin/pull/13 and the reason for it is that is needed for issue 12.

I was thinking I could continue by testing and doing the other issues, too, that have been solved for the 2.x version already but don't exist in 1.x branch yet. So I'm mostly following what has been done for the master branch.

Comment (KatseNarniaan) [reply src/main/java/io/openapitools/swagger/OutputFormat.java:45]: It is from #13, and it is also changing to use the swagger-core's object mapper, which has some defaults set in https://github.com/swagger-api/swagger-core/blob/1.5/modules/swagger-core/src/main/java/io/swagger/util/ObjectMapperFactory.java:

        mapper.setSerializationInclusion(JsonInclude.Include.NON_NULL);
        mapper.configure(SerializationFeature.FAIL_ON_EMPTY_BEANS, false);
        mapper.configure(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS, false);
        mapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);

Comment (langecode) [reply src/main/java/io/openapitools/swagger/OutputFormat.java:45]: Great - did not remember that one :-)

**Pull request merged**